pkg/schemaconv/openapi_test.go

```diff
@@ -53,8 +62,24 @@ func toPtrMap[T comparable, V any](m map[T]V) map[T]*V {
 }
 
 func normalizeTypeRef(tr *schema.TypeRef) {
-	if tr.Inlined == deducedDef.Atom {
-		// Deduplicate deducedDef (mostly for testing)
+	var untypedScalar schema.Scalar = "untyped"
```

Comment:
Why did you defined a couple at the top and this one here?

Reply:
eh cuz its not used anywhere else. the others are